HPXMLtoOpenStudio/resources/EPvalidator.rb

```diff
@@ -535,8 +535,7 @@ def self.run_validator(hpxml_doc)
       ## [WHType=Tank]
       '/HPXML/Building/BuildingDetails/Systems/WaterHeating/WaterHeatingSystem[WaterHeaterType="storage water heater"]' => {
         'FuelType[text()="natural gas" or text()="fuel oil" or text()="propane" or text()="electricity" or text()="wood"]' => one, # If not electricity, see [WHType=FuelTank]
-        'TankVolume' => one,
-        'HeatingCapacity' => one,
+        'TankVolume | HeatingCapacity' => zero_or_two,
```

Comment:
We can independently allow these to be provided, so you can just use zero_or_one for each of them. Will also need to update measure.rb code and the documentation accordingly.